ts/features/fci/components/SuccessComponent.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from "react";
+import I18n from "../../../i18n";
+import imageExpired from "../../../../img/wallet/errors/payment-expired-icon.png";
+import hourglass from "../../../../img/pictograms/hourglass.png";
+import { SignatureRequestDetailView } from "../../../../definitions/fci/SignatureRequestDetailView";
+import FciDocumentsScreen from "../screens/valid/FciDocumentsScreen";
+import ErrorComponent from "./ErrorComponent";
+import GenericErrorComponent from "./GenericErrorComponent";
+
+const now = new Date();
```

Comment:
Are you sure you want to create the Date here.
It will be never updated after app start.